adoc/MAIN.release-notes.adoc

```diff
@@ -19,6 +19,65 @@ Release notes usually only list changes that happened between two subsequent rel
 Certain important entries from the release notes documents of previous product versions may be repeated.
 To make such entries easier to identify, they contain a note to that effect.
 
+== Changes in 4.1.1
+
+{productname} can now be deployed with {tf} 0.12.
+
+=== Required Actions
+
+==== Upgrade terraform files and state
+
+If you can tear down your existing cluster, delete your cluster before
+upgrading to {tf} 0.12. You can then create a cluster following our
+documentation. It is the cleanest path.
+
+If you are using {tf} 0.11 and you cannot tear down your cluster, you will
+need to update your {tf} files (and states) in place for {tf} 0.12
+
+For this, enter your {tf} files/state folder and:
+
+* Migrate {tf} files with the automatic migration tool:
+  ``terraform 0.12upgrade``.
```

Comment:
A link to the official documentation would be handy.

Reply:
Agreed.

Reply:
Done.